src/my_sys.hpp

```diff
@@ -157,14 +158,14 @@ void DYING(const char *fmt, ...) CHECK_FORMAT_STR(printf, 1, 2);
 void CROAK(const char *fmt, ...) CHECK_FORMAT_STR(printf, 1, 2);
 void CROAK_CLEAN(const char *fmt, ...) CHECK_FORMAT_STR(printf, 1, 2);
 
-#define DIE(args...)                                                                                                 \
```

Comment:
By the way `(args...)` is also a GCC extension (https://gcc.gnu.org/onlinedocs/cpp/Variadic-Macros.html). I changed it to portable `__VA_ARGS__`.